postcode_lookup/template_sorter.py

```diff
@@ -162,7 +165,34 @@ def toc_label(self):
 
     @property
     def toc_id(self):
-        return "voter-registration"
+        return f"voter-registration-{self.timetable.registration_deadline}"
```

Comment:
`voter-registration` was already not a unique ID, and even less-so now.

I think this should make it unique

Reply:
I actually realised when I came back to this we need to account for both polling day and the registration deadline to ensure uniqueness

https://github.com/DemocracyClub/ec-postcode-lookup-pages/pull/108/commits/6d8956b223355bb469eab47d8c77e30953c5e766